src/bench/bench.cpp

```diff
@@ -59,20 +57,27 @@ void benchmark::BenchRunner::RunAll(const Args& args)
             continue;
         }
 
-        Bench bench;
-        bench.name(p.first);
-        if (args.asymptote.empty()) {
-            p.second(bench);
-        } else {
-            for (auto n : args.asymptote) {
-                bench.complexityN(n);
+        for (int i = 0; i < args.iters; ++i) {
+            if (i == 0 && args.iters > 1) {
```

Comment:
I do not think adding an iterations argument that work like that is a good idea. `nanobench` itself already has the ability to perform multiple measurements (called "epoch" in `nanobench`), in fact each benchmark is already measured 11 times. That way it is able to show a measurement error. The number of iterations in each of the measurements is determined automatically, based on the computer's clock accuracy.

If you get unstable benchmark results, the first thing to do should be to make sure the computer is really stable: no frequency scaling, no turbo, no other interfering programs. `nanobench` shows the warnings for good reason :slightly_smiling_face: 

If that doesn't help, make sure the actual benchmark itself is stable and actually always does the same (few randomness in it, better not have much allocations, threadding, locks, etc).

If that too doesn't help, you can e.g. increase the number of iterations with [minEpochIterations](https://nanobench.ankerl.com/reference.html#_CPPv4N6ankerl9nanobench5Bench18minEpochIterationsE8uint64_t). That's a bit problematic though because some benchmarks need a huge setting here, others a very low one. So generally it is probably better use [minEpochTime](https://nanobench.ankerl.com/reference.html#_CPPv4N6ankerl9nanobench5Bench12minEpochTimeENSt6chrono11nanosecondsE) and expose that setting in the arguments (probably as double value in seconds, e.g. like `-minEpochTime=0.5`)

Reply:
Yes. I've gone through these things. In practice, what I'm seeing people doing to share and compare results in PR reviews is run a benchmark repeatedly, for which the `iters=<n>` proposal here is a handy convenience.

Reply:
In any case, I'm happy to either remove `-quiet` and propose only `-iters=<n>`, or close and use these features for my own benchmarking use.